### PR TITLE
Support for Laravel 6+ and fixed error sending message. 

### PR DESCRIPTION
**Pull request by marvinhosea**
This PR fixes `Too few arguments to function` error when sending SMS with `\SMS::to('+254722000000')->message('Hi there')->send();`.

It also allows support for Laravel 6+.

🚧 WIP airtime payment.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,15 +25,13 @@
...
-        "predis/predis": "^1.1"
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.4",
+        "nesbot/carbon": "^2.27"
     },
     "require-dev": {
-        "mockery/mockery": "dev-master",
-        "phpunit/phpunit": "~5.7|~6.2"
+        "mockery/mockery": "^1.0",
+        "phpunit/phpunit": "^8.0"
```
Comment (SmoDav) [inline composer.json]: `~5.7|~6.2|~8.0`

File: composer.json [hunk 2]
```diff
@@ -25,15 +25,13 @@
...
     },
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.2",
-        "illuminate/support": "^5.0",
-        "nesbot/carbon": "~1.21",
-        "predis/predis": "^1.1"
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.4",
+        "nesbot/carbon": "^2.27"
```
Comment (SmoDav) [inline composer.json]: `illuminate/support` is needed for the Facades

**Review by SmoDav — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (marvinhosea) [reply composer.json]: Ok i will add it

**Comment by marvinhosea** [comment]
I have updated the requested changes with this commit 019ab5cfc4e7a670124e6700b6661408861d2e86 
kindly have a look

File: composer.json [hunk 3]
```diff
@@ -25,15 +25,14 @@
...
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.2",
-        "illuminate/support": "^5.0",
-        "nesbot/carbon": "~1.21",
-        "predis/predis": "^1.1"
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.4",
+        "nesbot/carbon": "^2.27",
+        "illuminate/support": "^6.16"
```
Comment (SmoDav) [inline composer.json]: to keep it backward compatible with previous Laravel versions, use
```
"illuminate/support": "^5.0 || ^6.0",
"nesbot/carbon": "^1.0 || ^2.0",
```

**Review by SmoDav — CHANGES_REQUESTED** [review]
Also revert all the file permissions to 644

File: composer.json (hunk 3 above)
Comment (marvinhosea) [reply composer.json]: Awesome insight, I have updated the requested changes with this commit 355f07868c90706838808f976c17c629306cc221

**Pull request merged**